API.md

```diff
@@ -91,9 +91,9 @@ An `HTTP 404 Not Found` status code is returned if the case with the specified I
 ]
 ```
 
-An 'HTTP 204 No Content' status code is returned is the aborted action with the specified ID has been rescheduled.
-An 'HTTP 404 Not Found' status code is returned when an invalid ID is specified.
-An 'HTTP 400 Bad Request' status code is returned when trying to rerun an action in a non aborted state.
+A 'HTTP 204 No Content' status code is returned is the aborted action with the specified ID that has been rescheduled.
```

Comment:
A 'HTTP 204 No Content' status code is returned `when` the aborted action with the specified ID that has been rescheduled.

Reply:
don't need that